src/server/worker/pipeline/transform/status.go

```diff
@@ -19,6 +19,7 @@ type Status struct {
 	jobID       string
 	datumStatus *pps.DatumStatus
 	cancel      func()
+	ImageID     string
```

Comment:
I don't think we should use the status to pipe this information through, we should probably just pipe it through as another parameter in the functions for now. The image id isn't needed by the status functionality.